### ISSUE TITLE
Product Query [3]: [Variation] `Newest Products` block

### ISSUE THREAD

**Issue by sunyatasattva** [woocommerce/woocommerce-gutenberg-products-block#7217]
Implement a “Newest Products” variation of the “Product Query” block. This variation should not show the order settings but should show at least the following settings:

* Category
* Stock status

This is blocked by #7218

**Comment by sunyatasattva** [comment]
Closing as won't do as of this comment: pdnLyh-2jZ-p2#comment-1685
